### PR TITLE
Usage example

### PR DESCRIPTION
**Pull request by jparisu**
Signed-off-by: jparisu <javierparis@eprosima.com>

### CODE REVIEW

File: docs/rst/getting_started/tutorial.rst [hunk 1]
```diff
@@ -6,55 +6,55 @@
...
 
-In order to show the Fast DDS Monitor running and monitoring a real DDS network, we are going to use a Fast DDS
+In order to show the *Fast DDS Monitor* running and monitoring a real DDS network, this tutorial uses a *Fast DDS*
 example to create a simple and understandable DDS network.
-The example proposed by this tutorial is using :code:`DDSHelloWorldExample` of Fast DDS repository.
-Be aware that the statistics module is not compiled and used by default by Fast DDS but it has to be specifically
-configured to run and send statistical data of an specific entity.
+The example proposed by this tutorial is using :code:`DDSHelloWorldExample` of *Fast DDS* repository.
+Be aware that the statistics module is not compiled and used by default by *Fast DDS* but it has to be specifically
+configured to run to send statistical data of an specific entity.
```
Comment (jparisu) [inline docs/rst/getting_started/tutorial.rst:20]: configured to send statistical data of an specific entity.

File: docs/rst/getting_started/tutorial.rst [hunk 2]
```diff
@@ -6,55 +6,55 @@
...
 
 In order to execute this minimum DDS scenario where each entity publish its statistical data, follow these steps:
 
-#. Compile Fast DDS library with CMake option :code:`FASTDDS_STATISTICS` to activate the statistics module.
-#. Compile Fast DDS library with CMake option :code:`COMPILE_EXAMPLES` to build the examples.
-#. Has an scenario with Fast DDS, Fast DDS Statistics Backend and Fast DDS Monitor built.
-#. Use Environmental Variable :code:`FASTDDS_STATISTICS` to activate the statistics writers in the DDS execution.
+#. Compile *Fast DDS* library with CMake option :code:`FASTDDS_STATISTICS` to activate the statistics module.
+#. Compile *Fast DDS* library with CMake option :code:`COMPILE_EXAMPLES` to build the examples.
+#. Have an working environment with *Fast DDS*, *Fast DDS Statistics Backend* and *Fast DDS Monitor* built.
```
Comment (jparisu) [inline docs/rst/getting_started/tutorial.rst:26]: #. Have a working environment with *Fast DDS*, *Fast DDS Statistics Backend* and *Fast DDS Monitor* built.

File: docs/rst/getting_started/tutorial.rst [hunk 3]
```diff
@@ -197,99 +200,87 @@ are related with the topics that we are not using will remain without data.
 Create Series Chart
 ===================
 
-In this section we are going to see how to graphically represent the data that is being monitored.
+This section describe how to graphically represent the data that is being monitored.
```
Comment (jparisu) [inline docs/rst/getting_started/tutorial.rst:203]: This section describes how to graphically represent the data that is being monitored.

File: docs/rst/getting_started/tutorial.rst [hunk 4]
```diff
@@ -197,99 +200,87 @@ are related with the topics that we are not using will remain without data.
...
 ---------------
 
-We are going to see how to represent the data that we are monitoring retrieving by the DDS entities.
-First of all we click in :ref:`display_data_button`.
-This will open a Dialog where we should choose one of the topics in which we want to see the data collected.
-For this case, we will choose :code:`DATA_COUNT`.
+This section explains how to represent the data being monitored and retrieved by the DDS entities.
+First of all, click in :ref:`display_data_button`.
+This will open a Dialog where you should choose one of the topics in which you want to see the data collected.
+The :code:`DATA_COUNT` han been choosen for this tutorial.
```
Comment (jparisu) [inline docs/rst/getting_started/tutorial.rst:211]: The :code:`DATA_COUNT` han been chosen for this tutorial.

**Review by jparisu — COMMENTED** [review]
Please, apply changes

**Review by jparisu — COMMENTED** [review]
LGTM

**Review by rsanchez15 — APPROVED** [review]
Approved in favour of https://github.com/eProsima/Fast-DDS-monitor/pull/62#pullrequestreview-685170872.

**Pull request merged**